api/v1/views/identity_membership.py

```diff
@@ -89,16 +90,16 @@ def delete(self, request, provider_uuid,
             return failure_response(
                 status.HTTP_404_NOT_FOUND,
                 "Identity does not exist.")
-        if not identity.can_share(user):
+        if not identity.can_share(request.user):
```

Comment:
Yes, I had the choice between defining an alias to request.user (as is done in L28) or just using request.  I choose the latter as for 2 function calls.  As jeremy points out L28 is in a different scope that is not accessible from here, so yes you do need something here.